coordinator/tests/services/translation/request/dispense/dispense-notification.spec.ts

```diff
@@ -17,6 +17,13 @@ import {
 } from "../../../../../src/services/translation/common/getResourcesOfType"
 import {ElementCompact} from "xml-js"
 import pino from "pino"
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
If there's an unused import, could we just remove it?

Reply:
mistake